README.md

```diff
@@ -4,4 +4,15 @@
 [![Travis-CI Build Status](https://travis-ci.org/mrc-ide/specio.svg?branch=master)](https://travis-ci.org/mrc-ide/specio)
 [![codecov.io](https://codecov.io/github/mrc-ide/specio/coverage.svg?branch=master)](https://codecov.io/github/mrc-ide/specio?branch=master)
 
-`specio` handles reading and parsing data from Spectrum PJNZ file into an R consumable format for use in the Estimation and Projection Package (EPP) for estimating and projecting HIV prevalence.
+`specio` handles reading and parsing data from Spectrum PJNZ file into an R consumable format for use in the [Estimation and Projection Package (EPP)](https://github.com/mrc-ide/eppasm) for estimating and projecting HIV prevalence.
+
+## Usage
+
+As well as the functions required for EPP `specio` also provides 2 more general functions for extracting population data and PLHIV data from the PJNZ file. These are `extract_population` and `extract_hiv_population`. See [website reference](https://mrc-ide.github.io/specio/reference/index.html) for usage and examples.
```

Comment:
Can you open a ticket for a usage vignette? (You already have the one on the internal format)

Reply:
Created that now. https://vimc.myjetbrains.com/youtrack/issue/mrc-273